components/new_channel_modal/new_channel_modal.tsx

```diff
@@ -8,6 +8,8 @@ import {Tooltip} from 'react-bootstrap';
 
 import classNames from 'classnames';
 
+import {lib as CryptoJS} from 'crypto-js';
```

Comment:
Could we use `crypto-js/core` here to only import the parts of the library that we need?

Reply:
This could be imported instead:
~~~ ts
import { random } from 'crypto-js/lib-typedarrays';
~~~
Which is the only function actually being used on [L116](https://github.com/mattermost/mattermost-webapp/pull/12169/files/8a1fb697e683041be70f95b94d01494d08a14791#diff-75a89e2e2f9c019418b2ae61429d34d36e3918eccd25b38b97bef97752ae5cd5R116).

Reply:
Yeah, that'd work if you could change that. That should reduce the amount of new code that we're needing to load on the client side

Reply:
Imported "random" function under a "cryptoJSRandomWordArray" alias for more clearance.